tests/components/StarRating/StarRating.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { StarRating } from '../../../src/Index';
+import { Mock } from 'coveo-search-ui-tests';
+import { IStarRatingOptions } from '../../../src/components/StarRating/StarRating';
+
+const CONTAINER_CSS_CLASS = 'CoveoStarRating';
+const STAR_CSS_CLASS = 'coveo-star-rating-star';
+const ACTIVE_STAR_CSS_CLASS = 'coveo-star-rating-star-active';
+const STAR_LABEL_CSS_CLASS = 'coveo-star-rating-label';
+
+const getActiveStars = (starRatingElement: HTMLElement) => {
+    const children = starRatingElement.children;
+    let numStars = 0;
+    let numActiveStars = 0;
+    for (let i = 0; i < children.length; i++) {
```

Comment:
Can you use `forEach` here instead of looping and using the index?

Reply:
Haha tried that at first but `children` isn't an array so it doesn't have a forEach. It's an HTMLCollection

Reply:
You can convert it to array with a `[...htmlCollection]` if necessary but that's not necessary I think :)

Reply:
Good to know!